src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
+    <link rel="stylesheet" href="./stars.css">
+    <link rel="stylesheet" href="./card.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="APPLE iMac 27"
+        class="card__product-image"
+      >
+
+      <h1 class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__review-number">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+
+        <p class="card__price-number">
+          $2,199
+        </p>
+      </div>
+
+      <div class="card__button">
+        <a
+          class="card__button-link"
+          href="#iMac"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
You don't need this div here and it creates a mess in css later. Try to simplify it and use `<a>` tag only